src/core/lib/security/transport/secure_endpoint.cc

```diff
@@ -143,7 +145,7 @@ struct secure_endpoint {
   grpc_core::MemoryAllocator::Reservation self_reservation;
   std::atomic<bool> has_posted_reclaimer;
   int min_progress_size;
-
+  grpc_slice_buffer tmp_frame_size_clip_buffer;
```

Comment:
I think this var should be renamed to something clearer or less specific.. something like protector_staging_buffer maybe?

Reply:
Done.